ui/src/pages/backoffice/Acquisition/Order/OrderDetails/PaymentInformation.js

```diff
@@ -7,18 +7,24 @@ export class PaymentInformation extends React.Component {
   render() {
     const order = this.props.order;
     const payment = order.payment;
-    if (!payment) return null;
     const leftTable = [
       {
         key: `Total (${order.grand_total_main_currency.currency})`,
         value: formatPrice(order.grand_total_main_currency),
       },
-      {
+    ];
+    if (
+      order.grand_total.currency !== order.grand_total_main_currency.currency
```

Comment:
It might be that librarian insert/misuse the totals and insert both with the same currency. Does it make sense to change this if and check if `grand_total` is defined instead?

Reply:
I kept both and indicated that the "main" one is the local currency with a popup explanation.